dev/user.clj

```diff
@@ -73,10 +73,15 @@
   (stop)
   (refresh-all :after 'user/re-start))
 
-(defn read-data [resource-path]
+(defn read-edn-data
+  [resource-path]
```

Comment:
It's dev code so not a big deal, but something like `read-edn-resource` might be more intuitive? Your change is already a big improvement, so 👍